data/css/modules/card/_media.scss

```diff
@@ -3,7 +3,7 @@
 
     &__caption {
         font-size: 20px;
-        font-family: Merriweather;
+        font-family: $body-font;
```

Comment:
You can use interactive rebase to fix your previous commit (basically by moving this commit below the original commit and set it as fixup)